### DISCUSSION TITLE
Repeatable Sections

### DISCUSSION THREAD

**Discussion by jakedewittdm** [vueform/builder#48] [Questions] [answered]
Would it be possible to add a section of components/elements that can repeat/duplicate. This is a feature in other form builders where someone might need to add a variable number of items. For example say you have a form that submits to an invoice, you might collecting the information of the products purchased, you would need to have someone for the user to add multiple products/name/qty/price etc. Is there anyway to do this with VueForm Builder?

**Comment by adamberecz** [comment, marked as answer]
Absolutely! In the [Developer version](https://builder.vueform.com/demo?v=dev) it can be found under Structure / List or Nested list, in  [Simple version](https://builder.vueform.com/demo?v=simple) it's Page / Repeat element or Repeat group.

**Comment by jakedewittdm** [comment]
Awesome thank you!
